docs/package_templates/autotools_package/all/conandata.yml

```diff
@@ -6,9 +6,7 @@ sources:
     - "https://mirror2.net/package-1.2.0.tar.gz"
     sha256: "________________________________________________________________"
   "1.1.0":
-    url:
-    - "https://mirror1.net/package-1.1.0.tar.gz"
-    - "https://mirror2.net/package-1.1.0.tar.gz"
+    url: "https://mirror2.net/package-1.1.0.tar.gz"
```

Comment:
To provide an example for newcomers of the more common non-list format for the URLs.